packages/validation-errors/src/ValidationErrors.tsx

```diff
@@ -59,9 +55,13 @@ function UniquenessError(props: UniquenessErrorProps) {
     [props.localeCode, props.defaultLocaleCode, contentTypesById]
   );
 
+  let conflicting: any[] = []
+  if ('conflicting' in props.error) {
+    conflicting = props.error.conflicting
+  }
```

Comment:
Any reason for moving away from the one-liner `props.error.conflicting?.map((entry) => entry.sys.id) || [];`?

Reply:
TIL